packages/html-manager/src/htmlmanager.ts

```diff
@@ -32,6 +34,16 @@ export class HTMLManager extends ManagerBase {
       },
       0
     );
+
+    this._viewList = new Set<DOMWidgetView>();
+    window.addEventListener('resize', () => {
+      this._viewList.forEach(view => {
+        MessageLoop.postMessage(
+          view.lmWidget,
+          Widget.ResizeMessage.UnknownSize
```

Comment:
Likely this will also need `LuminoWidget.Widget` instead of just `Widget` like in the code below